nexys4ddr-setup.tex

```diff
@@ -200,14 +200,28 @@ \section{Preparing microSDHC card}
 
 The MEGA65 requires an SDHC card of between 4GB and 64GB capacity.  Some SDXC cards may work, however, this is not officially supported.
 
-Preparation steps for the Nexys4 board's SD card share much in common with the steps needed for real MEGA65 hardware, and as such, it is worth having a look over the \nameref{cha:configuring} chapter if you ever need details.
+Preparation steps for the Nexys4 board's SD card share much in common with the
+steps needed for real MEGA65 hardware, and as such, it is worth having a look
+over
+\ifdefined\printmanual
+the {\bf MEGA65 Book}
+\else
+\bookvref{cha:configuringyourmega}
+\fi
+if you ever need details.
```

Comment:
In relation to all the changes in this file, the `nexys4ddr-setup.tex` chapter isn't a part of the user-guide, so I'm thinking we don't need all these \printmanual checks?

Reply:
It is in the big book and in `mega65-developer-guide.pdf`. In the latter it looks like:

![grafik](https://user-images.githubusercontent.com/67829169/210107413-83445420-baea-4a21-9e7c-f445b7d4f996.png)